webpack.prod.config.js

```diff
@@ -93,4 +94,15 @@ config.plugins = [
   }),
 ];
 
+/**
+ * The loader in the default config references a different version of
+ * MiniCssExtractPlugin and will break if we don't swap it with the one used in
+ * this file.
+ *
+ * TODO: This is a very brittle. We should improve this in the future by either
+ * defining the whole of the webpack config here or impementing some better
+ * configuration modification functionality in frontend-build.
+ */
+config.module.rules[2].use[0] = MiniCssExtractPlugin.loader;
```

Comment:
The most objectionable part of this file. The loader in the default config references a different version of MiniCssExtractPlugin and will break if we don't swap it with the one used in this file.

Reply:
It might be worth rebuilding the rules tree from scratch here, rather than relying on indices, yeah.  If we touch it in frontend-build, it's pretty likely to bust

Reply:
True, though then all those dev deps need to be managed by this repo. I'd prefer us to have a better way to manipulate configs. In the meantime I'm not sure what the right way to go is

Reply:
Updated with webpack-merge